OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -26,15 +26,15 @@ public class CombatDebugOverlay : IPostRender, INotifyDamage
 	{
 		readonly DeveloperMode devMode;
 
+		readonly HealthInfo health;
 		Lazy<AttackBase> attack;
 		Lazy<IBodyOrientation> coords;
-		Lazy<Health> health;
```

Comment:
It is not removing the laziness from a (lazy loaded) trait; rather, it is replacing a trait with its trait info which is initialized before actor creation starts.